### ISSUE TITLE
unknown string error in in built file

### ISSUE THREAD

**Issue by Om-patel2398** [spyder-ide/spyder#9498]
### What steps will reproduce the problem?

i am trying to run my program when this unknow error occured 
in file:- lib\xml\etree\ElementTree.py line 598 in parse
self._root = parser._parse_whole(source)
file:- "<string>", line unknown
ParseError: not well-formed (invalid token): line 44, column 88

cant solve

### Traceback
```python-traceback
  File "D:\ana\a3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 1686, in go_to_error
    self.edit_goto.emit(osp.abspath(fname), int(lnb), '')
ValueError: invalid literal for int() with base 10: ''
```

## Versions

* Spyder version: 3.3.4 
* Python version: 3.7.3
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.2 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.2 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.24.2 (OK)
numpy >=1.7       :  1.16.4 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29.2 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0     :  7.5.0 (OK)
matplotlib >=2.0.0:  3.1.0 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by ccordoba12** [comment]
Please post a simple program that generates the error you posted above so we can reproduce this in our side. Thanks!

**Comment by ccordoba12** [comment]
This was fixed in Spyder 3.3.5, which will be available soon in Anaconda.
